labgraph/lsl_node/lsl_sender_node.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+from pylsl import StreamOutlet, StreamInfo
+
+from ..graphs import Config, Node, Topic, subscriber
+from ..util.logger import get_logger
+from ..lsl_node import LSLMessage
+
+logger = get_logger(__name__)
+
+
+class LSLSenderConfig(Config):
+    name: str
```

Comment:
Shall we use stream_name and signal_type instead?

Reply:
Done